drivers/mil_blueview_driver/launch/example.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-    <node pkg="mil_blueview_driver" type="mil_blueview_driver" name="test_blue_view" >
+    <node pkg="mil_blueview_driver" type="blueview_driver" name="test_blue_view" >
```

Comment:
We did but in the cmake the target name of the executable is still blueview_driver.